R/Model-class.R

```diff
@@ -2508,8 +2491,8 @@ DALogisticLogNormal <- function(npiece = 3,
     datamodel = datamodel,
     priormodel = priormodel,
     modelspecs = modelspecs,
-    sample = c("alpha0", "alpha1", "lambda"),
-    datanames = c("nObs", "y", "x", "u", "Tmax")
+    datanames = c("nObs", "y", "x", "u", "Tmax"),
```

Comment:
you just changed the order of `datanames` and `sample`? that is not really necessary. Like we don't have an alphabetic ordering of slots or so.
can leave as is now but just for next time, only do the necessary formating.

Reply:
Yes, just wanted to keep `datanames` and `datanames_prior` closer to JAGS model stuff (as input), and sample is just output, so moved it to the end.